contracts/IErrors.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+interface IErrors {
+    // auth
+    error NotOwner();
+    error NotBackend();
+    error NotNode();
+    error NotSFC();
+    error NotDriver();
+    error NotDriverAuth();
+    error NotContract();
+    error NotAuthorized();
+
+    // initialization
+    error ContractInitialized();
```

Comment:
This should be called `InvalidInitialization` to be consistent with OpenZepelin's Initializable.
Also the "ContractInitialized" name is misleading, as it sounds like the initialization was successful which is not true.
Please keep this in the Initializable contract.